lib/kubernetes-deploy/kubernetes_resource/pod.rb

```diff
@@ -120,5 +140,69 @@ def display_logs
 
       @already_displayed = true
     end
+
+    class Container
```

Comment:
Don't spend too much time on it since it has a good integration coverage anyway 👌